crates/edr_napi/index.d.ts

```diff
@@ -637,6 +637,8 @@ export interface SolidityTestRunnerConfigArgs {
    * None.
    */
   includeTraces?: IncludeTraces
+  /** The configuration for the Solidity test runner's observability */
+  observability: ObservabilityConfig
```

Comment:
@frangio based on your TS experience, would you prefer this to be `observability?: ObservabilityConfig`?

Reply:
Yes, if all of the subobject fields are optional I'd prefer to omit the field rather than to have to write `observability: {}`. In this case there's also a backwards compatibility argument, though I don't think we consider that an important concern atm.